shared/prop-types/location-type.js

```diff
@@ -0,0 +1,9 @@
+import { PropTypes } from 'react';
```

Comment:
I like the idea of a prop-types folder. Perhaps we can call it shared prop types. If we are doing this, are there any other shared proptypes that can be moved here?